2-mandatory/1-create-functions.js

```diff
@@ -44,7 +50,13 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(numbers) {
+  return numbers.map((number) => {
+    if (number > 100) {
+      number = 100;
```

Comment:
This bit isn't right. Can you think why? You are looping through array of numbers and then reassigning 100 to each element in return.